pos_order_return/models/product_template.py

```diff
@@ -10,4 +10,5 @@ class ProductTemplate(models.Model):
 
     pos_allow_negative_qty = fields.Boolean(
         string='Allow Negative Quantity on PoS',
+        default=True,
```

Comment:
Why this default change?

Reply:
First, to avoid default annoying behaviors or tests fails. It's a nices feature to have, but wherever we got this module in an instance we finally had to set this setting to True by default.

Reply:
OK

Reply:
I would say that it will be nice to have this feature in another module (separate functionality).

Reply:
@chienandalu : Could you just change the readme of configuration . 

Replace : 

> In some cases, you may want to let the possibility to allow negative quantity

by 

> In some cases, you may want to **block** the possibility to allow negative quantity ... etc...